src/attributes/classes.js

```diff
@@ -23,9 +39,7 @@ jQuery.fn.extend( {
 			} );
 		}
 
-		if ( typeof value === "string" && value ) {
-			classes = value.match( rnothtmlwhite ) || [];
```

Comment:
87,169 bytes is the size of the master
87,258 bytes is the size of my local

I am running grunt on my local every time I hit save. The grunt checks passes. I wonder why they are failing at your end.